.packit.yaml

```diff
@@ -116,6 +116,7 @@ jobs:
       - fedora-39
       - fedora-38
       - epel-9
+      - epel-10
```

Comment:
I suspect packit will complain by filing an upstream issue until epel-10 is released. But feel free to try this out if you like. LGTM with that caveat.